### PR TITLE
Add slider components

### LINKED ISSUE #472: Add Slider/range components
**Scope**: All mapers

**Description**

Slider is one of basic input types and it's commonly used across webs https://www.w3schools.com/tags/att_input_type_range.asp

I think we should add it as an another component types

**Schema**

```jsx
{
   components: componentsTypes.SLIDER,
   name: 'radius',
   min: 0,
   max: 100,
   step: 1,
   label: 'Select radius of your pizza delivery (km)'
}
```

**References**

MUI https://material-ui.com/components/slider/#slider
PF3 https://patternfly-react-pf3.surge.sh/?path=/story/patternfly-react-forms-and-controls-slider--slider
PF4 https://patternfly.github.io/v4/design-guidelines/usage-and-behavior/data-input#slider *coming soon*

What do you think @Hyperkid123 ?

### PR DESCRIPTION
**Pull request by rvsia**
closes #472 

TODO:
- [x] Renderer constant
- [x] MUI slider
- [x] PF3 slider
- [x] documentation

### CODE REVIEW

**Comment by Hyperkid123** [comment]
@rvsia can you add just a regular slider for PF4? So we have something in place until there is an existing component?

**Comment by Hyperkid123** [comment]
I also see "1" character when previewing the PF3 slide:
screenshot-localhost_3000-2020 05 06-11_49_01

Can you check where is it coming from?

**Comment by Hyperkid123** [comment]
It seems that the value is popping out (probably has some z-index value)
screenshot-localhost_3000-2020 05 06-11_50_24

**Comment by rvsia** [comment]
@Hyperkid123 Yeah, the styling of the PF3 slider is wrong, but I did my best and copied all sliders CSS to our mastermind CSS file. It helped a lot, but it's not perfect. :-/

Yeah, I can add a placeholder to PF4.

**Comment by Hyperkid123** [comment]
@rvsia can you override it so it does not mess up the docs?

**Comment by rvsia** [comment]
@Hyperkid123 

1. added a PF4 slider

image

2. the number in PF3 is hidden

File: packages/react-renderer-demo/src/app/src/doc-components/mui-slider.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+**Props**
+
+Slider is wrapped in a form group, so it accepts all [form group props](/renderer/component-api#formgroupwrappedcomponents).
+
+|Props|Type|Description|
+|-----|----|-----------|
+|after|node|Node rendered after the slider|
+|before|node|Node rendered before the slider|
+|FormFieldGridProps|object|Props passed to FormFieldGrid component|
```
Comment (Hyperkid123) [inline packages/react-renderer-demo/src/app/src/doc-components/mui-slider.md:11]: Can we add links to the corresponding API of the MUI component? The rest of the MUI components does have this link.

Comment (rvsia) [reply packages/react-renderer-demo/src/app/src/doc-components/mui-slider.md]: done

**Pull request merged**